actions/registerRealm.ts

```diff
@@ -41,14 +49,37 @@ import { withSetRealmAuthority } from '@models/withSetRealmAuthority'
 */
 
 /**
- * The default amount of community tokens with 0 supply
+ * The minimum amount of community tokens to create governance and proposals, for tokens with 0 supply
  */
-const DEF_COMMUNITY_TOKENS_W_0_SUPPLY = 1000000
+export const MIN_COMMUNITY_TOKENS_TO_CREATE_W_0_SUPPLY = 1000000
 
 /**
  * The default amount of decimals for the community token
  */
-const COMMUNITY_MINT_DECIMALS = 6
+export const COMMUNITY_MINT_DECIMALS = 6
+
+/**
+ * Try to get the associated token account of an owner to a mint id
+ * @param connection
+ * @param owner
+ * @param mint
+ */
+async function tryGetAta(
```

Comment:
Why is this function duplicated here? Can we use this one https://github.com/solana-labs/governance-ui/blob/2639a59f203d274bbb3a81361fab69b630aa274b/utils/validations.tsx#L48 ?

It really should be moved to tokens.tsx to be consistent

Reply:
fixed